source/glkernel/include/glkernel/noise.h

```diff
@@ -35,6 +35,16 @@ void normal(tkernel<V<T, P>> & kernel, T mean, T stddev);
 template <typename T, glm::precision P, template<typename, glm::precision> class V>
 void normal(tkernel<V<T, P>> & kernel, const V<T, P> & mean, const V<T, P> & stddev);
 
+enum class PerlinNoiseType { Standard, Cloud, CloudAbs, Wood, Paper };
```

Comment:
i disagree, this should remain in noise.h for now - the idea is to collect every feature by namespace. since this is a header only library i would like to reduce the number of includes for now. if all those headers are becoming `to big` we could discuss a review of our header/namespace design ...